### ISSUE TITLE
Error: Cannot find module '@truffle/db'

### ISSUE THREAD

**Issue by Sptz** [trufflesuite/truffle#3896]
- [ ] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

Installing Truffle yields errors in requesting Python as well as Visual Studio. This is both on a year old Windows 10 installation as well as a fresh one. Post installation, running any truffle command apart from _init_ results in the error message in the title. 

## Actual Results

Error message from typing _truffle test_ for example:

```
Error: Cannot find module '@truffle/db'
Require stack:
- C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\cli.bundled.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (internal/modules/cjs/helpers.js:88:18)
at Object.168060 (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\external "@truffle\db":1:1)
at __webpack_require__ (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\webpack\bootstrap:18:1)
at Object.577135 (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\workflow-compile\index.js:12:28)
at __webpack_require__ (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\webpack\bootstrap:18:1)
at Object.952422 (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\core\lib\testing\Test.js:9:25)
otstrap:18:1)
at prepareConfigAndRunTests (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\core\lib\commands\test\prepareConfigAndRunTests.js:3:16)
at Object.run (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\core\lib\c at processTicksAndRejections (internal/process/task_queues.js:93:5)
at Command.run (C:\Users\Pedro\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\core\lib\command.js:136:1)
Truffle v5.2.1 (core: 5.2.0)
Node v14.16.0
```

Log output when installing truffle: https://pastebin.com/gLFbxV2V

Log output when installing truffle after downloading and installing Python 3.9: https://pastebin.com/aB4CFUFb

## Environment

* Operating System: Windows 10 Pro
* Ethereum client:
* Truffle version (`truffle version`): 5.2.1
* node version (`node --version`): 14.16.0
* npm version (`npm --version`):

**Comment by gnidan** [comment]
Working on this! Should have a fix out tonight. Sorry about this!

**Comment by gnidan** [comment]
This should be fixed now in [v5.2.2](https://github.com/trufflesuite/truffle/releases/tag/v5.2.2).

Closing this optimistically, but let us know if the problem persists; always happy to do another hotfix :)

Thanks all!
